classes/fields/file.php

```diff
@@ -95,6 +101,24 @@ public function options() {
 					'browse' => __( 'Media Library', 'pods' ),
 				),
 			),
+			static::$type . '_upload_dir'             => array(
+				'label'      => __( 'Upload directory', 'pods' ),
+				'default'    => 'wp',
+				'type'       => 'pick',
+				'data'       => array(
+					'wp'      => __( 'WordPress Default', 'pods' ) . ' (/yyyy/mm/)',
+					'uploads' => __( 'Custom directory within the default uploads directory', 'pods' ),
+				),
+				'depends-on' => array( static::$type . '_uploader' => 'plupload' ),
+				'dependency' => true,
+			),
+			static::$type . '_upload_dir_custom'     => array(
+				'label'       => __( 'Custom upload directory', 'pods' ),
+				'description' => __( 'Magic tags allowed', 'pods' ),
+				'depends-on'  => array( static::$type . '_upload_dir' => 'uploads' ),
+				'default'     => apply_filters( "pods_form_ui_field_{$type}_upload_dir_custom", '' ),
```

Comment:
```suggestion
				/**
				 * Allow filtering the custom upload directory used.
				 *
				 * @since 2.7.28
				 *
				 * @param string @default_directory The custom upload directory to use by default for new fields.
				 */
				'default'     => apply_filters( "pods_form_ui_field_{$type}_upload_dir_custom", '' ),
```